cron/reset.py

```
@@ -0,0 +1,36 @@
+    from rush import quota, throttle
+    from rush.limiters import periodic
+    from rush.stores import redis as redis_store
+
+
+@app.function(
+    schedule=modal.Cron("0 8 * * *"),  # PST time. Every day at 8:00
+    retries=modal.Retries(max_retries=3),
+)
+def clear_limit():
+    # Duplicated from config.py
+    REDIS_URL = os.environ["REDIS_URL"]
```

Ruff

Variable `REDIS_URL` in function should be lowercase

(N806)